### ISSUE TITLE
OS X issues

### ISSUE THREAD

**Issue by kbariotis** [andrewchambers/godothecorrectthing#3]
I can't make it work on OS X(High Sierra):

```
+ set -f
+ editor=subl
++ guesscwdwithmagic
++ cwd=/Users/Bariotis
+++ xdotool getactivewindow getwindowname
Error: Can't open display: (null)
Failed creating new xdo instance
++ wintitle=
+ cwd=
```

I've tried [exporting `DISPLAY`](https://stackoverflow.com/questions/24464294/xdotool-doesnt-work-via-ssh) but I keep getting the same error.

I liked your idea though :)

**Comment by andrewchambers** [comment]
Yeah, somebody will need to write custom tools to get the clipboard and the window title on mac to make it work. I don't think mac uses x11 for everything, so xdotool and xclip won't do the job properly.

It might be possible with something like https://github.com/Hammerspoon/hammerspoon, though I have never used or owned a mac, so I won't be able to help. Perhaps somebody will fork and make a mac version.
